src/projects.js

```diff
@@ -0,0 +1,11 @@
+const Project = (title) => {
+  const todoList = [];
+
+  const id = Date.now().toString();
+
+  return {
+    title, id, todoList,
+  };
+};
+
+export default Project;
```

Comment:
### [proposal]
Normally, this module should manage the entire logic behind projects
- addition
- deletion
- returning list of projects
It's not just about creating the object

check this [example](https://github.com/shloch/TODO_app/blob/master/src/projectModule.js)

Reply:
I ask for help from a TSE since I'm working alone in this project, and I took his approach to the project and todo model, but thanks for the proposal.